setup.py

```diff
@@ -11,7 +11,7 @@
 
 # Don't import quandl module here, since deps may not be installed
 sys.path.insert(0, os.path.join(os.path.dirname(__file__), 'quandl'))
-from version import VERSION
+from version import VERSION  # NOQA
```

Comment:
there's a link to the issue in the file. Also talked about it here (i think): https://quandl.slack.com/archives/core-product-team/p1458223827000678